applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp

```diff
@@ -416,6 +416,36 @@ bool CheckWakeCondition(const Element& rElement, const double& rTolerance, const
     return wake_condition_is_fulfilled;
 }
 
+template <int Dim, int NumNodes>
+double ComputeLocalMachSquaredDerivative(
+    const array_1d<double, Dim>& velocity,
+    const double& local_mach_number,
```

Comment:
```suggestion
    const array_1d<double, Dim>& rVelocity,
    const double rLocalMachNumber,
```

Reply:
I think @philbucher actually meant: const double& rLocalMachNumber

To follow the [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#arguments): "Write arguments with a UpperCamelCase format. First input arguments, then output arguments. If the argument is passed as a reference, add 'r' at the beginning of the name. If the argument is a pointer, add 'p' at the beginning of the name."

Reply:
nope, I meant `const double rLocalMachNumber`

there is no reason to use a reference here, only confuses the reader

afaik both double and ref to double have a size of 64bits

Reply:
why then using r if it is not a reference?

Reply:
ah
sure, without the `r`

I was focused on the `&` :)

Reply:
Ok!

> there is no reason to use a reference here, only confuses the reader
> 
> afaik both double and ref to double have a size of 64bits

that's actually a good point, I didn't know it